pkg/master/reconcilers/lease_test.go

```diff
@@ -417,15 +418,14 @@ func TestLeaseEndpointReconciler(t *testing.T) {
 	for _, test := range reconcileTests {
 		fakeLeases := newFakeLeases()
 		fakeLeases.SetKeys(test.endpointKeys)
-		clientset := fake.NewSimpleClientset()
+		var endpoints []runtime.Object
```

Comment:
I'm not sure this is technically related to the OrDie changes, but it is simpler...